platform.md

```diff
@@ -672,8 +749,12 @@ Usage:
     - MUST have the working directory set to the value of `<app>`.
     - MUST contain the following labels
         - `io.buildpacks.lifecycle.metadata`: see [lifecycle metadata label](#iobuildpackslifecyclemetadata-json)
-        - `io.buildpacks.project.metadata`: the value of which SHALL be the json representation `<project-metadata>`
+        - `io.buildpacks.project.metadata`: the value of which SHALL be the json representation of `<project-metadata>`
         - `io.buildpacks.build.metadata`: see [build metadata](#iobuildpacksbuildmetadata-json)
+        - **If** image extensions were used to extend the run image and `<extended>/<image config>` has the label `io.buildpacks.rebasable` set to `true`:
```

Comment:
This might be redundant, as we already say that image config labels are preserved.